conjur/logout/logout_controller.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+"""
+LogoutController module
+
+This module is the controller that facilitates all logout actions
+required to successfully logout
+"""
+
+# Builtins
+import logging
+import sys
+
+# Internals
+from conjur.utils import Utils
+
+# pylint: disable=too-few-public-methods
+class LogoutController:
+    """
+    LogoutController
+
+    This class represents the Presentation Layer for the LOGOUT command
+    """
+    logout_logic = None
+    ssl_verify = None
+
+    def __init__(self, ssl_verify, logout_logic):
+        self.ssl_verify = ssl_verify
+        if self.ssl_verify is False:
+            Utils.insecure_message()
```

Comment:
See my comment on `login_controller` about this.

Reply:
removed b/c of PO requirement